scripts/config/templates/nginx.conf.erb

```diff
@@ -220,14 +220,20 @@ http {
     location ~ "(*UTF8)^\/(garages|parking-spaces)\/([A-z0-9\-\p{L}'%]+)$" {
       return 301 https://$host/regions/$2/$1$is_args$args;
     }
-    
+
     # Redirect for single dash dlps
     location ~* "(*UTF8)^\/[A-z0-9\-\p{L}'%]*--[A-z0-9\-\p{L}'%]*-(garages|parking-spaces|long-term-parking|monthly-parking|self-storage|driveway-parking)$" {
       mruby_set $locations_path /app/bin/config/lib/ngx_mruby/locations_single_path.rb cache;
       return 301 https://$host/locations/$locations_path$is_args$args;
     }
   <% end %>
 
+  # Redirect location DLPs to server-side rendered frontend
+  # The /build directory is also needed because that's where the server-rendered JS comes from
+  location ~ ^/((build|locations)/.*) {
+    proxy_pass <%= ssr_frontend_host %>/$1;
```

Comment:
I'm confused at the group match here (`$1`). Wouldn't that grab the contents of the first group `(build|locations)`?

Reply:
Great question. Regular expressions track the group match number based on where the first parentheses begin, so the first group match is actually the `((build|locations)/.*)` one, and the `(build|locations)` match becomes `$2`. The regex test in the description has more info on that.

Reply:
ah, I see now